src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -1162,6 +1163,89 @@ private static FormulaValue Refresh(EvalVisitor runner, EvalVisitorContext conte
         public static FormulaValue PatchRecord(IRContext irContext, FormulaValue[] args)
         {
             return CompileTimeTypeWrapperRecordValue.AdjustType((RecordType)FormulaType.Build(irContext.ResultType._type), (RecordValue)MutationUtils.MergeRecords(args).ToFormulaValue());
+        }
+
+        public static async ValueTask<FormulaValue> Summarize(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, FormulaValue[] args)
+        {
+            if (args[0] is BlankValue)
+            {
+                return new BlankValue(irContext);
+            }
+
+            if (args[0] is not TableValue tableValue)
+            {
+                return CommonErrors.RuntimeTypeMismatch(irContext);
+            }

```

Comment:
This would be overriding any errors that are passed to the function, and we shouldn't. This expression should return a Div0 error:

    Summarize(
        If(1/0<2, [{a:1,b:2},{a:1,b:3}]),
        a,
        CountRows(ThisGroup) As Count
    )